auditors/capabilities/fix.go

```diff
@@ -63,3 +62,24 @@ func removeCapabilityFromAddList(container *k8stypes.ContainerV1, capability str
 
 	container.SecurityContext.Capabilities.Add = added
 }
+
+type fixMissingSecurityContextOrCapability struct {
+	container *k8stypes.ContainerV1
+}
+
+func (f *fixMissingSecurityContextOrCapability) Plan() string {
+	return fmt.Sprintf("Adds capability to '%s' with drop set to ALL", f.container.Name)
```

Comment:
I'm a bit confused by the `Adds capability to '%s'`. This sounds like we're adding a capability. Maybe it can just be something like `Sets capability drop list  to ["ALL"]`?

Reply:
That was a bad copy pasta from another fix! 🙈 Thanks for catching it! 🙏🏽